torch/jit/__init__.py

```diff
@@ -1096,9 +1121,11 @@ def forward(self, input):
                     output = self.linear(output)
                     return output
 
-            scripted_module = torch.jit.script(MyModule())
+            scripted_module = torch.jit.script(MyModule(2, 3))
```

Comment:
This changed because It errored previously ?

Reply:
Yeah